modules/aws_base/input_schema.aws_base.json

```diff
@@ -0,0 +1,54 @@
+{
```

Comment:
to reduce maintenance while we have both implementation, would it be possible to reuse the existing json file? (`modules/aws_base/aws-base.json`)

Reply:
_We could add support to allow referencing the old file to the new YAML format._
that sounds like a good idea, and this would address the concern about maintaining two almost identical json files